kong/db/declarative/import.lua

```diff
@@ -122,7 +128,13 @@ end
 
 
 local function get_current_hash()
-  return lmdb.get(DECLARATIVE_HASH_KEY)
+  local v, err = lmdb_get(LMDB_KONG_VERSION_KEY)
+  if v ~= KONG_VERSION or err then
+    ngx.log(ngx.WARN, "current Kong v", KONG_VERSION, " mismatch cache v", v)
+    return nil, err
```

Comment:
`err`  would be `nil` here? will there be some issues for the caller func?

Reply:
This seems something we could check with an unit test. 

@chronolaw could you add some testing to this PR?

Reply:
Returning `err` is a protecting action, for possible lmdb incident.

Reply:
Could you give a hint for testing? It need upgrade kong.

Reply:
I checked the places where `get_current_hash` is used, and it seems that none of them handle the second return value of `err`.

Reply:
> I checked the places where `get_current_hash` is used, and it seems that none of them handle the second return value of `err`.

Yes, but it should be compatible with the original `lmdb_get`.

Reply:
Okay, just a doubt, I have no problem now.